src/sentry/api/urls.py

```diff
@@ -998,6 +999,11 @@
         r"^teams/",
         include(
             [
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/$",
```

Comment:
We have an endpoint that is similar to this, `OrganizationTeamsEndpoint`, which is under `organizations/<slug>/teams`. Since this new endpoint is similar, let's put it under the same path. Something like `organizations/<slug>/user-teams`

Reply:
sounds good